README.md

```diff
@@ -1 +1,33 @@
-# kitcat
\ No newline at end of file
+# KitCat
+
+*Sometimes schema migrations are just not enough.*
+
+Data migration framework written in plain Ruby. It is pretty generic, since it does not depend on any web framework.
+
```

Comment:
I think we ned to put somewhere, Whether in this readme or another doc that we created this Gem as we needed a solution for MongoDB  data migrations. It would then make the examples that use mongo make a bit more sense